src/jsxc.lib.xmpp.chatState.js

```diff
@@ -130,9 +130,10 @@ jsxc.xmpp.chatState.onPaused = function(ev, jid) {
    }
 
    if (usersComposing.length === 0) {
-      var duration = parseFloat(el.css('transition-duration'));
+      var durationValue = el.css('transition-duration') || '0';
```

Comment:
Why not ` || '0s'`? I think that's the more or less offiecial default, according to https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration#Summary